test/lib/examples/hasExample.test.ts

```
@@ -1,7 +1,7 @@
 import { describe, expect, it } from 'vitest'
 import { hasExample } from '../../../src/lib/examples/hasExample'
 import { specWithCircularRef, specWithCircularRefAndExample } from '../../testsConstants'
-import { getSchemaUi } from '../../../src/lib/getSchemaUi'
+import { getSchemaUi } from '../../../src/lib/processOpenAPI/getSchemaUi'
```

ESLint

[error] Expected "../../../src/lib/processOpenAPI/getSchemaUi" to come before "../../testsConstants".

(perfectionist/sort-imports)